includes/html/api_functions.inc.php

```diff
@@ -1250,10 +1250,17 @@ function add_edit_rule(Illuminate\Http\Request $request)
         $disabled = 0;
     }
 
+    $invert_map = $data['invert_map'];
```

Comment:
Is this just a much longer way to write:
```php
$invert_map = (int) ($data['invert_map'] ?? 0);
```

Note: a lot of this code is missing isset or nullable checks.  Probably causes errors when fields are missing in the request.

Reply:
or maybe like this `($data['invert_map'] ?? 0) ? 1 : 0`

Reply:
or maybe like this: `$invert_map = (int) ! empty($data['invert_map');`

Reply:
Hello,
You may be right I don't know all the syntax in PHP. I just copied the existing code for the $disabled variable.
I don't know which syntax is the best in term of performance, though.
Do you want me to update those if conditions for $disabled and $invert_map?

I tested both variables with your new syntax and it is working.

Reply:
Your second proposition is causing the API to crash and it is not easy to troubleshoot for me.